packages/cardano-services/src/Metadata/types.ts

```diff
@@ -0,0 +1,11 @@
+import { Cardano } from '@cardano-sdk/core';
+
+export interface MetadataService {
```

Comment:
I think it would be great to have a bit more documentation for the public interface of the modules. for example, what purpose/functionality a metadata service provides?

Reply:
Added some doc comments in [9271def](https://github.com/input-output-hk/cardano-js-sdk/pull/297/commits/9271def01fc6cef2f05c20636733d1bafb82b37c)